### PR TITLE
Adding legacy ABAC for 1.6

### PR DESCRIPTION
**Pull request by Q-Lee**
**What this PR does / why we need it**: optionally enable legacy abac for 1.6, so existing users aren't broken.

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by Q-Lee** [comment]
cc @cjcullen

**Comment by Q-Lee** [comment]
cc @deads2k - heads up, we need to be able to optionally enable legacy ABAC in 1.6. It will remain off unless it's explicitly set.

**Comment by deads2k** [comment]
> cc @deads2k - heads up, we need to be able to optionally enable legacy ABAC in 1.6. It will remain off unless it's explicitly set.

I remember @cjcullen had a plan here.  I'd like to see his comments before merge.

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1010,6 +1010,21 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+
+  # Create the ABAC file only if it's explicitly requested.
+  if [[ -n "${ENABLE_LEGACY_ABAC}" ]]; then
```
Comment (deads2k) [inline cluster/gce/gci/configure-helper.sh]: I'd like a rename to `ENABLE_LEGACY_ABAC_16_ONLY` and and an issue to delete this in 1.7.

Comment (cjcullen) [inline cluster/gce/gci/configure-helper.sh]: I'm fine with that (and we can have the deprecation policy discussion in that issue).

Comment (cjcullen) [inline cluster/gce/gci/configure-helper.sh]: This also needs to be `{ENABLE_LEGACY_ABAC_16_ONLY:-}` so that it doesn't bail if that var isn't set.

Comment (jcbsmpsn) [inline cluster/gce/gci/configure-helper.sh]: Original author unavailable so this PR is updated and continued on https://github.com/kubernetes/kubernetes/pull/42197

**Comment by ethernetdan** [comment]
Removing PR from 1.6 milestone since code freeze has begun:
* If this is a bug fix, please label as `kind/bug` and reapply the milestone.
* If this is a feature for 1.6, an [exception](https://github.com/kubernetes/features/blob/master/EXCEPTIONS.md) should be filed for it before 11:00PM PST on 3/2. Note that most exceptions to code freeze are not accepted.
* If this is a feature for 1.7, feel free to add the 1.7 milestone to the PR.

/cc @kubernetes/release-team @kubernetes/kubernetes-release-managers

**Comment by calebamiles** [comment]
This PR also does not have justification for why it should be considered a release blocker. Removing this label. Please do not add a `release-blocker` label without consulting with the [release team](https://github.com/kubernetes/features/blob/master/release-1.6/release_team.md). Thanks!

**cc: @ethernetdan, @kubernetes/kubernetes-release-managers**

**Pull request closed without merging**